composer.json

```diff
@@ -4,13 +4,17 @@
     "keywords": ["event", "loop", "async", "interop"],
     "license": "MIT",
     "require": {
-        "php": ">=5.5.0"
+        "php": ">=5.5.0",
+        "async-interop/event-loop-implementation": "^0.5"
```

Comment:
We can't require an implementation in this package, as it makes it impossible to run the tests without at least adding an implementation to `require-dev`.